pyproject.toml

```diff
@@ -49,6 +49,7 @@ click = "^7.1.2"
 docker = "^5.0.0"
 ply = "^3.11"
 python = "^3.8"
+pytest = "^6.2.3"
```

Comment:
This should be a development dependency not a production dependency, right?

Reply:
Oh no I was trying to add test coverage and that snuck in. Whoops.

Reply:
Easy enough to fix.  Sorry my review wasn't complete like I was expecting it to be.

Reply:
No worries. Ill probably do a roundup of these and do a QA fix PR if necessary.